migrations/sql/V2019.05.31_1753__UsersManagePermission.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO "orderlyweb_permission" VALUES ("users.manage")
```

Comment:
We could just stick this in the prev migration with the other permissions, since nothing is deployed yet. Don't mind too much either way, just a thought.